Assets/Scripts/UI/PauseMenu.cs

```diff
@@ -72,97 +94,139 @@ private void Start()
         {
             BuildSettingsUI(x);
         }
         currentPanel = 0;
     }
-    
+
     private void Update()
     {
         if (InputManager.GetButtonDown(PlayerInput.PlayerButton.Pause, player))
         {
-            if (!isPaused && !pauseMenu.activeInHierarchy)
+            if (!isPaused && !main.activeInHierarchy)
             {
                 Pause();
             }
             else if(panels[currentPanel].activeInHierarchy)
             {
                 panels[currentPanel].SetActive(false);
-                pauseMenu.SetActive(true);
+                main.SetActive(true);
                 currentPanel = 0;
 
             }
             else if(settings.activeInHierarchy)
             {
                 settings.SetActive(false);
-                pauseMenu.SetActive(true);
+                main.SetActive(true);
             }
-            else
+            else if(SceneManager.GetActiveScene().name != "Title")
             {
-                ResumeGame();
+                ResumeGame();                
             }
-            EventSystem.current.SetSelectedGameObject(pauseMenu.transform.GetChild(0).gameObject);
+            EventSystem.current.SetSelectedGameObject(main.transform.GetChild(0).gameObject);
         }
     }
+    #region Title Methods
+    public void StartGame()
+    {
+        var latest = GameManager.latestUnlocked;
+        //SceneManager.LoadScene( (latest >= GameManager.totalLevels || latest < 1) ? 1 : GameManager.latestUnlocked);
+        SceneManager.LoadScene(1); // START ON THE FIRST LEVEL FOR DEMO BUILD
+    }
+    #endregion
+
+    #region Menu Methods
     public void Pause()
     {
         isPaused = true;
-        camControl.enabled = false;
-        playerMovement.enabled = false;
+        if (SceneManager.GetActiveScene().name != "Title")
+        {
+            camControl.enabled = false;
+            playerMovement.enabled = false;
+            playerUI.enabled = false;
+        }
         Time.timeScale = 0;
-        playerUI.enabled = false;
         HUD.SetActive(false);
-        pauseMenu.SetActive(true);
+        main.SetActive(true);
         Cursor.visible = true;
         Cursor.lockState = CursorLockMode.None;
-        
+
     }
     public void ResumeGame()
     {
         isPaused = false;
-        pauseMenu.SetActive(false);
+        main.SetActive(false);
         Time.timeScale = 1;
         Cursor.visible = false;
         Cursor.lockState = CursorLockMode.Locked;
-        camControl.enabled = true;
-        playerMovement.enabled = true;
-        playerUI.enabled = true;
+        if (SceneManager.GetActiveScene().name != "Title")
+        {
+            camControl.enabled = true;
+            playerMovement.enabled = true;
+            playerUI.enabled = true;
+        }
         HUD.SetActive(true);
     }
     public void MainMenu()
     {
         SceneManager.LoadScene("Title");
     }
+    #endregion
+
+    #region Shared Methods
```

Comment:
Just for future reference any "shared methods" should be abstracted away to a separate class entirely.
The title menu shouldn't have access to pause menu specific functions and visa versa. 
I wouldn't bother changing this on this project, they already seem to be tightly coupled but it is important to think about when structuring future projects.